frontend/src/components/AcitveMembership/ActiveMembership.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+// @material-ui/core components
+import { Grid } from "@material-ui/core";
+// core components
+import Divider from "@material-ui/core/Divider";
+
+interface Keyable {
+    [key: string]: string
+}
+
+interface Props {
+    planType: string,
+    expiry: string,
+    classes: Keyable
+};
+
+const ActiveMembership: React.FC<Props> = (props) => {
```

Comment:
You can use plain old typescript here
`const ActiveMembership = (props: Props) => {`